### ISSUE TITLE
Map crash

### ISSUE THREAD

**Issue by Oremuss** [rtr-nettest/open-rmbt#9]
Hello again me with another issue or actually two separate issues.

First isssue is that I have tested app in emulator and everything is working. (except map, but thats that second issue). Now when I put that in my mobile phone with android 6, it works aswell, but when I put the same .apk to the android 9 phone, it doesnt work and says it cannot find server. Any idea with this weirb behaving?

Also as I mentioned everytime I click on Map, the app always crash. It dount matter if its in phone or in emultaor, it crashers everytime.

And also I noticed that measured is speed is showing me around 30 Mbit per second but mz real speed is 10 times higher, around 300-500 Mbit per second. I testet original app from RTR = netztest from goole play and it shows correct values, around 350 Mbit. Is it slow because of somehow I set something wrong?

**Comment by cproof** [comment]
Hello,

> Edit#2: Found solution.
> Starting with Android 9 (API level 28), cleartext support is disabled by default.

Thank you for this. This concerns only your test setup, as the connections are unencrypted? Or is this a general problem? In any case we can add that to the `README`.

> And also I noticed that measured is speed is showing me around 30 Mbit per second but my real speed is 10 times higher, around 300-500 Mbit . I testet original app from RTR-netztest downlaoded from google play and it shows correct values, around 350 Mbit.

How is your laptop connected to the test server? Maybe your VPN / Proxy / test server upload connection speed is slowing traffic down.

**Comment by Oremuss** [comment]
> Thank you for this. This concerns only your test setup, as the connections are unencrypted? Or is this a general problem? In any case we can add that to the `README`.

Yes my connection is not encrypted so I guess it is the problem. Anywaz it would be nice to have that in README :)  

> How is your laptop connected to the test server? Maybe your VPN / Proxy / test server upload connection speed is slowing traffic down.
Also true , I am connected via VPN so I am positive that it is the problem.

But main issue is still here, when I try Map tab, whole app crash. Everytime I try, emulator in Android Studio and real app too.

**Comment by cproof** [comment]
> But main issue is still here, when I try Map tab, whole app crash. Every time I try, whether it is emulator in Android Studio or real app. Could there be possibly something wrong with settings?

Could you obtain and add the logcat output on the crash? (e.g. from the Android emulator)
